pkg/languagedetection/detector.go

```diff
@@ -39,6 +39,14 @@ var exactMatches = map[string]languageFromCLI{
 	"node": {name: languagemodels.Node},
 
 	"dotnet": {name: languagemodels.Dotnet},
+
+	"rails":   {name: languagemodels.Ruby},
+	"ruby":    {name: languagemodels.Ruby},
+	"rubyw":   {name: languagemodels.Ruby},
+	"sidekiq": {name: languagemodels.Ruby},
+	"irb":     {name: languagemodels.Ruby},
+	"puma":    {name: languagemodels.Ruby},
+	"puma:":   {name: languagemodels.Ruby},
```

Comment:
Where does `jruby` fit here, is it `ruby` or java? Example: `jruby rails server`

Reply:
😅 I think it's ruby, since the actual code is written in ruby? Let me ask @dineshg13